README.md

```diff
@@ -18,8 +18,22 @@ Development Installation
 5. See the app at http://localhost:8024! See OpenTripPlanner at http://localhost:9090.
 6. Running `npm run gulp-watch` from `/opt/app/src` will automatically collect static files together when changes are detected for Django template consumption.
 
-Building AMIs via Packer
+Building AMIs
 ------------------------
 1. Make a production group_vars file (similarly to how is described above with development). Make sure production is set to true, and also specify an app_username, which should be set to: ubuntu
-2. Install Packer from: https://www.packer.io/downloads.html
-3. In the project directory, run: packer build -var 'aws_access_key=YOURaccessKEYhere' -var 'aws_secret_key=YOURsecretKEYhere' deployment/packer/cac.json
+2. Install Packer from: `https://www.packer.io/downloads.html`
```

Comment:
Would also probably be nice/good idea to have packer installed in one of the VMs during development to make it easy to just run this from within the VM since we already have a [packer role](https://github.com/azavea/ansible-packer)